pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -559,3 +567,15 @@ func (r *ReconcilePravegaCluster) isRollbackTriggered(p *pravegav1alpha1.Pravega
 	}
 	return false
 }
+
+func (r *ReconcilePravegaCluster) compareKeys(cm1 *corev1.ConfigMap, cm2 *corev1.ConfigMap) bool {
```

Comment:
This should be a method that returns intersection of set A and Set B. Return a `string` and `error` where the `string` returns all overlapping keys and error is non-nil if we failed to compute the intersection.